### DISCUSSION TITLE
Directus is able to manage an external db?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#5263] [Archive]
Is Directus able to manage data and attach api to them from an already in place and Ester al db?

**Comment by rijkvanzanten** [comment]
I don't know what you mean by "and Ester al db?", but yes, Directus can manage an existing database.

Btw, [the Discord chat](https://directus.chat) (https://directus.chat) is a better place for these shorter explorative questions 👍🏻

**Reply by biapar** [reply]
External ... (t9 maleficus)
